src/index.ts

```diff
@@ -113,13 +115,20 @@ export class MondayRedisService extends BaseConnector {
   }
 
   private serialize(value: any, title: string) {
+    if (!value) {
```

Comment:
Should be "value === undefined". For example, !0 is also true

Reply:
it was to also fix the problem for null. But good point, if we set an integer to 0, it's gonna have a nasty side effect. I'll fix it with a check undefined or null as suggested